docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/core.entity_view_display.node.cgov_mini_landing.default.yml

```diff
@@ -128,6 +129,13 @@ content:
     third_party_settings: {  }
     weight: 7
     region: content
+  field_mlp_page_style:
+    type: list_default
+    label: above
+    settings: {  }
+    third_party_settings: {  }
+    weight: 16
+    region: content
```

Comment:
What are the other fields here? IDK if this should set to show, or hide by default. Default would be used when we don't have a view mode defined. I don't see this showing up under that circumstance.